src/routes/index.js

```diff
@@ -3,14 +3,14 @@
 const flatten = arrayOfArrays => [].concat([], ...arrayOfArrays);
 
 const joinRoutes = routes => flatten(
-    routes.map(route => {
-      if (!Array.isArray(route)) {
-        return [route];
-      }
+  routes.map(route => {
+    if (!Array.isArray(route)) {
+      return [route];
+    }
 
-      return route;
-    }),
-  );
+    return route;
+  }),
+);
```

Comment:
This is [Prettier](https://github.com/prettier/prettier) fighting with the pre-commit hook linter